Source/WTF/wtf/text/StringImpl.h

```diff
@@ -538,6 +545,42 @@ class StringImpl : private StringImplShape {
 
 public:
     void assertHashIsCorrect() const;
+
+    // Table originally from the SpiderMonkey JavaScript engine.
+    static constexpr char escapedFormsForJSON[0x100] = {
```

Comment:
I wonder if we should define it as

```
static WTF_EXPORT_PRIVATE const char escapedFormsForJSON[0x100];
```

in the header, and define this in cpp to avoid materializing this table repeatedly in various frameworks (WebKit.framework, WebCore.framework, and JavaScriptCore.framework).

Reply:
I thought of that, but it can’t be used from the functions called from the constexpr `StaticStringImpl` constructor if it’s defined that way. Maybe we can find a way to have it both ways, and the non-constexpr uses can all share a single table.